imagetest/cmd/manager/main.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
-	imagevalidation "github.com/GoogleCloudPlatform/guest-test-infra/imagetest/test_suites/image_validation"
```

Comment:
this is not necessary, i was leaving image_validation this way to show how the package name does not have to match the directory name

Reply:
Revert the change.